src/App.js

```diff
@@ -39,15 +39,14 @@ export default function App() {
         <StylesProvider injectFirst>
           <CssBaseline />
           <BrowserRouter>
-            <HandleRedirectTo />
             <ScrollToTop />
             <AppBar />
             <Switch>
               <Route exact path="/" component={HomePage} />
               <Route exact path="/alert_signup" component={HomePage} />
               <Route
                 exact
-                path="/compare/:compareShareId?"
+                path="/compare/:sharedComponentId?"
```

Comment:
`sharedComponentId` is now a "well-known" param used by e.g. the "useSharedComponentParams()" hook.